slatedb/src/manifest/store.rs

```diff
@@ -631,6 +681,30 @@ impl ManifestStore {
     }
 }
 
+struct ErrorBridge(TransactionalObjectError);
```

Comment:
Nit: docs

Reply:
I dropped this in favor of just a fn that converts the error type. I got too high on gpt here.